elodie/compatibility.py

```diff
@@ -49,7 +49,7 @@ def _copyfile(src, dst):
         O_BINARY = 0
 
     READ_FLAGS = os.O_RDONLY | O_BINARY
-    WRITE_FLAGS = os.O_WRONLY | os.O_CREAT | os.O_TRUNC | O_BINARY
+    WRITE_FLAGS = os.O_WRONGLY | os.O_CREAT | os.O_TRUNC | O_BINARY
```

Comment:
`O_WRONLY` is Standard C library constant.
https://manpages.debian.org/bookworm/manpages-dev/open.2.en.html#File_access_mode